packages/dart/noports_core/lib/src/srvd/isolates/relay_worker.dart

```diff
@@ -0,0 +1,181 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:isolate';
+
+import 'package:at_utils/at_logger.dart';
+import 'package:meta/meta.dart';
+import 'package:noports_core/src/srvd/isolates/types.dart';
+import 'package:noports_core/sshnp_foundation.dart';
+import 'package:noports_core/src/srvd/relay_auth_verifiers.dart';
+import 'package:noports_core/src/srvd/srvd_session_params.dart';
+
+typedef RelayWorkerRequestHandler = Future<dynamic> Function(IIRequest);
+
+abstract class RelayWorker implements RelayAuthVerifyHelper {
```

Comment:
Superclass with useful stuff for comms between main isolate and the 443 isolate and the ephemeral PortPair isolates